bambou/error_test.go

```diff
@@ -50,47 +48,10 @@ func TestError_NewError(t *testing.T) {
 func TestError_Error(t *testing.T) {
 
 	Convey("Given I create a new Error", t, func() {
-		e := NewError(42, "a big error")
+		e := NewBambouError("Title", "Description")
 
-		Convey("Then Error() output should be should be '<Error: 42, message: a big error>'", func() {
-			So(e.Error(), ShouldEqual, "<Error: 42, message: a big error>")
-		})
-	})
-}
-
-func TestError_FromJSON(t *testing.T) {
```

Comment:
Shouldn't we be testing VSD Error as well? Throwing a fake VSD error and verify it is handled correctly?